src/icons/social/bluesky.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 57">
+    <path fill="currentColor"
+        d="M13.873 3.805C21.21 9.332 29.103 20.537 32 26.55v15.882c0-.338-.13.044-.41.867-1.512 4.456-7.418 21.847-20.923 7.944-7.111-7.32-3.819-14.64 9.125-16.85-7.405 1.264-15.73-.825-18.014-9.015C1.12 23.022 0 8.51 0 6.55 0-3.268 8.579-.182 13.873 3.805ZM50.127 3.805C42.79 9.332 34.897 20.537 32 26.55v15.882c0-.338.13.044.41.867 1.512 4.456 7.418 21.847 20.923 7.944 7.111-7.32 3.819-14.64-9.125-16.85 7.405 1.264 15.73-.825 18.014-9.015C62.88 23.022 64 8.51 64 6.55c0-9.818-8.578-6.732-13.873-2.745Z">
+    </path>
+</svg>
```

Comment:
Ran this through [SVGOMG](https://jakearchibald.github.io/svgomg/) to squeeze a few more bytes out of it (saves ~78 bytes gzipped):

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 57">
  <path fill="currentColor" d="M13.9 3.8A75.1 75.1 0 0 1 32 26.6v15.8c0-.3-.1 0-.4.9-1.5 4.5-7.4 21.8-21 8-7-7.4-3.8-14.7 9.2-17-7.4 1.4-15.7-.7-18-9A193 193 0 0 1 0 6.6C0-3.3 8.6-.1 13.9 3.9Zm36.2 0A75.1 75.1 0 0 0 32 26.6v15.8c0-.3.1 0 .4.9 1.5 4.5 7.4 21.8 21 8 7-7.4 3.8-14.7-9.2-17 7.4 1.4 15.7-.7 18-9A193 193 0 0 0 64 6.6c0-9.8-8.6-6.7-13.9-2.7Z"/>
</svg>
```